datapackage.json

```diff
@@ -56,27 +56,64 @@
                         "format": "default"
                     },
                     {
-                        "name": "Macomber Keywords",
-                        "type": "string",
+                        "name": "Earliest Attestation",
+                        "type": "integer",
+                        "format": "default",
+                        "description": "The earliest century in which the story appeared."
+                    },
+                    {
+                        "name": "Total Records",
+                        "type": "integer",
                         "format": "default"
                     },
                     {
-                        "name": "CSM Number",
+                        "name": "Total Paintings",
+                        "type": "integer",
+                        "format": "default"
+                    },
+                    {
+                        "name": "Hamburg ID",
                         "type": "string",
                         "format": "default"
                     },
+                    {
+                        "name": "CSM Number",
+                        "type": "integer",
+                        "format": "default",
+                        "constraints": {
+                            "minimum": 1,
+                            "maximum": 420
+                        }
+                    },
+                    {
+                        "name": "Poncelet Number",
+                        "type": "integer",
+                        "format": "default",
+                        "constraints": {
+                            "minimum": 1,
+                            "maximum": 1783
+                        }
+                    },
                     {
                         "name": "Clavis ID",
                         "type": "string",
+                        "format": "default",
+                        "constraints": {
+                            "pattern": "^CAe [1-9]\\d{3}$"
```

Comment:
I think the pattern should be `"^CAe [1-9]\d{3}$"`The double backslash occurs in all the regex in this JSON. Perhaps because of how it was encoded?